clintfidel/WEconnect
# feat(view-business-reviews):  users should be able to view all review…

#### What does this PR do?
This PR enables users to successfully view all reviews for a business in the application

#### Description of Task to be completed?
This PR ensures that a registered user is able to view all reviews for a business in the application
#### How should this be manually tested?
- visit application url [here](weconnectapp.herokuapp.com)
- Navigate to the signup page and create an account or Login if you already have an account
- You would be redirected to the all business page
- click on **view more** to view the business profile
- scroll to the button and you should see reviews

#### What are the relevant pivotal tracker stories?
[156789303](https://www.pivotaltracker.com/story/show/156789303)

#### Any background context you want to add?
N/A

#### Screenshots
![image](https://user-images.githubusercontent.com/26988297/38791406-638726ca-413f-11e8-9340-833400ce83fe.png)
